CMakeLists.txt

```diff
@@ -117,8 +117,16 @@ if (BUILD_TESTS)
   # which might be ON.
   set(INSTALL_GTEST OFF CACHE BOOL "Do _not_ install gtest" FORCE)
 
+  # Make sure that gtest/gmock is built as static libs
+  set(BUILD_SHARED_LIBS OFF CACHE BOOL "")
```

Comment:
Looks ugly:). Can we consider/try use these instead:
```
-DGTEST_CREATE_SHARED_LIBRARY=0 
-DGTEST_LINKED_AS_SHARED_LIBRARY=0 
```
I would like to avoid rewriting/changing widely used cmake-defined variables as much as possible.